Examples/ExampleMultiProjectIntegration/ExampleMultiProjectIntegration.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,662 @@
+				3289B3FE2BF955710053F2E4 /* Frameworks */,
+				3289B3FF2BF955710053F2E4 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = Subproject;
+			packageProductDependencies = (
+				3289B4102BF955DF0053F2E4 /* SafeDI */,
+			);
+			productName = Subproject;
+			productReference = 3289B4012BF955710053F2E4 /* Subproject.framework */;
+			productType = "com.apple.product-type.framework";
+		};
+/* End PBXNativeTarget section */
+
+/* Begin PBXProject section */
+		32756FDA2B24C042006BDD24 /* Project object */ = {
+			isa = PBXProject;
+			attributes = {
+				BuildIndependentTargetsInParallel = 1;
+				LastSwiftUpdateCheck = 1500;
+				LastUpgradeCheck = 1510;
+				TargetAttributes = {
+					32756FE12B24C042006BDD24 = {
+						CreatedOnToolsVersion = 15.0.1;
+					};
+					3289B4002BF955710053F2E4 = {
+						CreatedOnToolsVersion = 15.4;
+					};
+				};
+			};
+			buildConfigurationList = 32756FDD2B24C042006BDD24 /* Build configuration list for PBXProject "ExampleMultiProjectIntegration" */;
+			compatibilityVersion = "Xcode 14.0";
+			developmentRegion = en;
+			hasScannedForEncodings = 0;
+			knownRegions = (
+				en,
+				Base,
+			);
+			mainGroup = 32756FD92B24C042006BDD24;
+			packageReferences = (
+				3203E22C2BF94D3F0094A988 /* XCRemoteSwiftPackageReference "SafeDI" */,
+			);
+			productRefGroup = 32756FE32B24C042006BDD24 /* Products */;
+			projectDirPath = "";
+			projectRoot = "";
+			targets = (
+				32756FE12B24C042006BDD24 /* ExampleMultiProjectIntegration */,
+				3289B4002BF955710053F2E4 /* Subproject */,
+			);
+		};
+/* End PBXProject section */
+
+/* Begin PBXResourcesBuildPhase section */
+		32756FE02B24C042006BDD24 /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				32756FEE2B24C044006BDD24 /* Preview Assets.xcassets in Resources */,
+				32756FEA2B24C044006BDD24 /* Assets.xcassets in Resources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+		3289B3FF2BF955710053F2E4 /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXResourcesBuildPhase section */
+
+/* Begin PBXShellScriptBuildPhase section */
+		3203E2192BF7B4630094A988 /* Run SafeDITool */ = {
+			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Run SafeDITool";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "xcrun swift run --package-path $BUILD_ROOT/../../SourcePackages/checkouts/SafeDI --scratch-path $BUILD_DIR/SafeDITool-Release -c release SafeDITool --include ExampleMultiProjectIntegration Subproject --dependency-tree-output $PROJECT_DIR/ExampleMultiProjectIntegration/SafeDI.swift\n";
```

Comment:
This isn't what I'm recommending in the `README.md`, but it works well enough for a test in CI. However, we shouldn't waste CI resources building for release.

```suggestion
			shellScript = "xcrun swift run --package-path $BUILD_ROOT/../../SourcePackages/checkouts/SafeDI --scratch-path $BUILD_DIR/SafeDITool-Release SafeDITool --include ExampleMultiProjectIntegration Subproject --dependency-tree-output $PROJECT_DIR/ExampleMultiProjectIntegration/SafeDI.swift";
```